src/pytest_regressions/num_regression.py

```diff
@@ -73,11 +73,21 @@ def check(
 
         __tracebackhide__ = True
 
+        for k, obj in data_dict.items():
+            if not isinstance(obj, np.ndarray):
+                try:
+                    arr = np.atleast_1d(np.asarray(obj))
+                except:
```

Comment:
This `except/else` clause here does not seem necessary, please remove it.

Reply:
Done